src/test/resources/config/application.yml

```diff
@@ -43,15 +43,9 @@ spring:
       hibernate.cache.use_query_cache: false
       hibernate.generate_statistics: false
       hibernate.jdbc.time_zone: UTC
-  data:
-    elasticsearch:
-      cluster-name: docker-cluster
-      cluster-nodes: 127.0.0.1:9300
-      properties:
-        path:
-          logs: target/elasticsearch/log
-          data: target/elasticsearch/data
-          home: target/elasticsearch
+  elasticsearch:
+    rest:
+      uris: http://localhost:9225
```

Comment:
Port number change

Reply:
Elasticsearch now uses the HTTP port, this is correct